packages/components/Organisms/Publication/PublicationAuthorInformation.vue

```diff
@@ -14,16 +14,17 @@ export default {
     RplDescriptionList
   },
   props: {
-    author: Array,
+    author: String,
     date: String,
     copyright: String,
     locale: { default: 'en-au', type: String }
   },
   computed: {
     informationListing () {
       const terms = []
-      if (this.author && this.author.length > 0) {
-        terms.push({ term: pluralize('Author', this.author.length), description: this.author.join(', ') })
+      if (this.author) {
+        const numberOfAuthors = (this.author.match(/,/g) || []).length + 1
```

Comment:
Hey @yeniatencio, 

Going by the storybook example authors can be departments i.e. "Department of Premier and Cabinet", this means an author name could have a comma in it like "Department of Families, Fairness and Housing", which would lead to the incorrect number of authors i.e. pluralization. 

We could do this the opposite way and revert the PublicationAuthorInformation.vue so it accepts an array again then in the author mapping stuff make sure author is always an array.

```
const author = Array.isArray(authorsName) ? authorsName.map((author) => author.name) : [authorsName.name]
```

Reply:
Good call @lambry . I have updated my pr for your review. Thanks